rockcraft.yaml

```diff
@@ -134,3 +134,4 @@ services:
       SNAPCRAFT_BUILD_ENVIRONMENT: host
       # put libexec on the path, for craftctl
       PATH: /usr/libexec/snapcraft:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+      PYTHONPATH: /usr/lib/python3.12/site-packages
```

Comment:
it might be better to setup a sitecustomize for this to avoid having this leaked into your parts being built when using the python plugin